organization/board-of-directors-meetings/2022-02-09-minutes.md

```diff
@@ -0,0 +1,25 @@
+---
+type: Resource
+---
+
+# Board meeting February 10th 2022
+
+1. Review action items:
+   1. Action: Eric to calculate the effect of the suggested salary policy on the burn rate
+      1. Unchanged: Blocked on needing a meeting with the bookkeeper
+   2. Action: Ben will revise the statement of support or partnership agreement based on the one sent to Montreal for approval by the board
+   3. Action: Eric and Boris to renegotiate with Multiple Ventures BV for services in 2022
+      1. Follow up: Eric and Boris to follow up with Multiple Ventures BV
+   4. Action: Ben with help from Eric and Boris will work on a good framing for our work in North America in Quarter 1
+      1. Follow up: Ben will propose a request for proposal and send this to Eric and Boris
```

Comment:
Just checking that "propose a request for proposal" is correct and that it is not meant to be "propose a request" or "request a proposal"

Reply:
A "request for quote" (an RFQ) and "request for proposal" are jargon, yes.